de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5446010_sys_FRESH_334_Modify_BPProduct_Primary_Key.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE c_bpartner_product DROP CONSTRAINT c_bpartner_product_pkey;
```

Comment:
* when changing the PK DDL, please also check, if there should be an accompanying change int the AD
* i think that
  - introducing c_bpartner_product_id as single-column key and 
  - adding a UC constraint that includes "where isactive='Y' would be better and cleaner
wdyt? if in doubt, i recomment to consult  @teosarca